datashuttle/tui/configs.py

```diff
@@ -446,6 +456,17 @@ def setup_configs_for_a_new_project(self) -> None:
         with the new project.
         """
         project_name = self.query_one("#configs_name_input").value
+        # Add validation before proceeding
+        if not project_name.strip():
```

Comment:
Just as a general tip, if you have a piece of code you are using in one place, you can create a function and place the code there, then call it from multiple places. The benefit of this is that if you need to change the code in future, you only have to do it in one place.